openmetadata-spec/src/main/resources/json/schema/entity/data/query.json

```diff
@@ -0,0 +1,91 @@
+{
+  "$id": "https://open-metadata.org/schema/entity/data/Query.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Query",
+  "description": "This schema defines the type to capture the table's queries.",
+  "type": "object",
+  "javaType": "org.openmetadata.schema.entity.data.Query",
+  "javaInterfaces": ["org.openmetadata.schema.EntityInterface"],
+  "properties": {
+    "id": {
+      "description": "Unique identifier of the query.",
+      "$ref": "../../type/basic.json#/definitions/uuid"
+    },
+    "query": {
+      "description": "query.",
```

Comment:
description should be "SQL Query definition"

Reply:
done